conda-forge.yml

```diff
@@ -3,3 +3,8 @@ azure:
   settings_win:
     pool:
       vmImage: windows-2022
+build_platform:
```

Comment:
Please don't touch this file on staged-recipes. You can do this in thr feedstock later on.